buildchain/buildchain/targets/local_image.py

```diff
@@ -84,22 +84,22 @@ def show(_task: types.Task) -> str:
 
     def _build_actions(self) -> List[types.Action]:
         """Build a container image locally."""
-        build_cmd = [
-            config.DOCKER, 'build',
-            '--tag', self.tag,
-            '--file', self.dockerfile,
-        ]
-        for arg, value in self.build_args.items():
-            build_cmd.extend(['--build-arg', '{}={}'.format(arg, value)])
-        build_cmd.append(self.dockerfile.parent)
-
-        actions : List[types.Action] = [build_cmd]
+        actions : List[types.Action] = []
+
+        builder = DockerBuild(
+            tag=self.tag,
+            path=self.dockerfile.parent,
+            dockerfile=self.dockerfile,
+            buildargs=self.build_args
+        )
+        actions.append((builder, [], {}))
```

Comment:
Let's not pass empty overrides when not needed:

```suggestion
        actions.append(builder)
```

Reply:
See above, will do.